anchor/network/src/discovery.rs

```diff
@@ -304,14 +364,35 @@ impl Discovery {
                                 (enr, None)
                             })
                             .collect();
+                        debug!(peers = ?results, "Discovery query completed");
                         return Some(results);
                     }
                     Err(e) => {
                         warn!(error = %e, "Discovery query failed");
                     }
                 }
             }
-            _ => {
+            QueryType::Subnet(queries) => {
+                let subnets_searched_for: Vec<SSVSubnet> =
+                    queries.iter().map(|query| query.subnet).collect();
+
+                match query.result {
+                    Ok(r) if r.is_empty() => {
+                        debug!(subnets_searched_for = ?subnets_searched_for, "Grouped subnet discovery query yielded no results.");
```

Comment:
```suggestion
                        debug!(
                            subnets_searched_for = ?subnets_searched_for,
                            "Grouped subnet discovery query yielded no results.",
                        );
```

just a nitpick as `rustfmt` doesn't split long macro invocations